cookbooks/burnman.prm

```diff
@@ -0,0 +1,140 @@
+# A cookbook for using adiabatic conditions and a gravity 
+# profile generated by a mineral physics toolkit and read in
+# through an ascii data file. In this example, material properties
+# are based on a Birch-Murnaghan equation of state. 
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 2.63e8
+
+
+subsection Material model
+  set Model name = ascii reference profile
+
+  subsection Ascii reference profile
+    set Thermal viscosity exponent = 10.0
+    set Viscosity prefactors = 1.0, 0.1, 1.0, 10.0
+  end
+
+  subsection Ascii data model
+    set Data file name = isentrope_properties.txt
+    set Data directory = $ASPECT_SOURCE_DIR/data/adiabatic-conditions/ascii-data/
```

Comment:
since the file name is relative to the directory, maybe reverse the order of parameters here? That is also what you do further down